man/coap_session.txt.in

```diff
@@ -26,6 +26,7 @@ coap_session_get_proto,
 coap_session_get_state,
 coap_session_get_tls,
 coap_session_get_type,
+coap_session_is_encrypted,
```

Comment:
As you may have discovered, you will need to add in an entry line into man/Makefile.am
````
        @echo ".so man3/coap_session.3" > coap_session_get_type.3
+      @echo ".so man3/coap_session.3" > coap_session_is_encrypted.3
        @echo ".so man3/coap_string.3" > coap_delete_bin_const.3
````
I would also prefer to see this entry at the end of the list, after the set/get functions.